### PR TITLE
Fix startup speed

### LINKED ISSUE #313: Massive activation time regression
It seems that one of the recent versions has introduced a massive `require` into the activation logic again as there is a ~280 ms load on my machine now for a new window (~1.4s during initial Atom load).

### PR DESCRIPTION
**Pull request by steelbrain**
Fixes #313

Before
screen shot 2017-05-20 at 9 28 07 pm

After
screen shot 2017-05-20 at 9 26 04 pm

### CODE REVIEW

**Comment by steelbrain** [comment]
Here's the latest
screen shot 2017-05-22 at 1 15 19 am

File: lib/panel/dock.js [hunk 1]
```diff
@@ -14,6 +10,11 @@ class PanelDock {
   constructor(delegate: Object) {
     this.element = document.createElement('div')
     this.subscriptions = new CompositeDisposable()
+
+    const React = require('react')
```
Comment (Arcanemagus) [inline lib/panel/dock.js]: Although `require` caching _works_, it's probably better to not rely on it exclusively and make the references to these at the file level.

Comment (steelbrain) [inline lib/panel/dock.js]: This is only ever constructed once 😉

Comment (Arcanemagus) [inline lib/panel/dock.js]: Exactly my point, it's a rare occurrence so forcing the caching is better than relying on the automatic caching.

Comment (steelbrain) [inline lib/panel/dock.js]: Done in https://github.com/steelbrain/linter-ui-default/commit/74f9f139d2a85cef72e74408e3c26e6defdddee2

**Pull request merged**